src/Input/InputKeys.cpp

```diff
@@ -61,6 +79,9 @@ static constexpr struct string_to_key string_to_key[] = {
   { _T("F18"), KEY_F18 },
   { _T("F19"), KEY_F19 },
   { _T("F20"), KEY_F20 },
+
+
+
```

Comment:
Why do you suggest to add three empty lines here, and what does this have to do with this overall change?

Reply:
will remove it.